alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java

```diff
@@ -38,4 +40,20 @@ public BaseHandlerStd(final AliasHelper aliasHelper) {
       CallbackContext callbackContext,
       ProxyClient<KmsClient> proxyClient,
       Logger logger);
+
+  /**
+   * Perform the final propagation delay to make sure the latest
+   * changes to the alias are available throughout the region.
```

Comment:
Seems like this function is being used for more than just aliases

Reply:
Nope, it's just being used for aliases. There is another version of this function in the Key package, that is used for the Key handlers.